release-notes/major9/09.17.0/valtimo-backend-libraries.md

```diff
@@ -26,6 +26,10 @@ The following features were added:
   
   More information on plugin secrets can be found [here](/extending-valtimo/plugin/custom-plugin-definition.md#plugin-properties).
 
+* **Support for Keycloak resource roles**
+
+  In addition to the realm roles Valtimo can now also include resource roles for a specified client. Add the client 
```

Comment:
In addition to the realm roles Valtimo... -> In addition to realm roles, Valtimo...